web/js/containers/sidebar/smart-handoff-modal.js

```diff
@@ -0,0 +1,157 @@
+            management for all EOSDIS system components (DAACs, Tools, Services). Your Earthdata login also helps the EOSDIS program
+            better understand the usage of EOSDIS services to improve user experience through customization of tools and improvement
+            of services. EOSDIS data are openly available to all and free of charge except where governed by international agreements.
+          </p>
+        </div>
+      </div>
+      )}
+
+      <div id="toggle-more-info" onClick={() => toggleInfo(!showMoreInfo)}>
+        <h2><span>{showMoreInfo ? 'Hide Info' : 'Show More Info'}</span></h2>
+      </div>
+
+      <div id="layer-info">
+        <h1> Selected layer to download: </h1>
+        <a href={cmrSearchDetailURL} target="_blank" rel="noopener noreferrer">
+          <p id="layer-name">{`${title}`}</p>
+          <p id="layer-mata-data">{`${subtitle} (${date})`}</p>
+        </a>
+      </div>
+
+
+      <div className="button-group">
+        <Button
+          onClick={() => goToEarthDataSearch()} // Need to pass reference of current state of boundaries
+          id="continue-btn"
+          text="Continue"
+          className="red"
+        />
+      </div>
+
+      <div id="checkbox-footer">
+        <div id="checkbox-footer-container">
+          <Checkbox
+            id="hide-eds-checkbox"
+            name="hide-eds"
+            onCheck={() => hideModal()}
+            checked={false}
+            color="gray"
+            aria-label="Do not show the Earthdata Search message again."
+            label="Do not show this message again."
+          />
+        </div>
+      </div>
+
+    </div>
+
+  );
+}
+
+const hideModal = () => {
+  const { HIDE_EDS_WARNING } = safeLocalStorage.keys;
+  const shouldHideWarning = safeLocalStorage.getItem(HIDE_EDS_WARNING);
+  if (!shouldHideWarning) safeLocalStorage.setItem(HIDE_EDS_WARNING, true);
+  else safeLocalStorage.removeItem(HIDE_EDS_WARNING);
+};
+
+
+/**
+ * Handle type-checking of defined properties
+ */
+SmartHandoffModal.propTypes = {
+  extentCoords: PropTypes.object,
+  goToEarthDataSearch: PropTypes.func,
+  selectedDate: PropTypes.instanceOf(Date),
+  selectedLayer: PropTypes.object,
+};
+
+/**
+ * ReactRedux; used for selecting the part of the data from the store
+ * that the Smarthandoff component needs. This is called every time the
+ * store state changes.
+ * @param {*} state | Encapsulates the entire Redux store state.
+ * @param {*} ownProps | Data from SmartHandoff that is used to retrieve data from the store.
+ */
+const mapStateToProps = (state, ownProps) => ({});
+
+/**
+ * React-Redux; used for SmartHandoff component to fire specific actions events
+ * @param {*} dispatch | A function of the Redux store that is triggered upon a change of state.
+ */
+const mapDispatchToProps = (dispatch) => ({
+
+});
+
+export default connect(
+  mapStateToProps,
+  mapDispatchToProps,
+)(SmartHandoffModal);
```

Comment:
Since you're not actually hooking up any state to props, or dispatching anything, you can remove the `mapStateToProps` and `mapDispatchToProps` definitions here and also remove the `connect()` call to wire this up to Redux.